docs/concepts/storage/persistent-volumes.md

```diff
@@ -573,68 +573,51 @@ parameters:
-  diskformat: zeroedthick
-```
+       ```yaml
+       kind: StorageClass
+       apiVersion: storage.k8s.io/v1
+       metadata:
+         name: fast
+       provisioner: kubernetes.io/vsphere-volume
+       parameters:
+         diskformat: zeroedthick
+       ```
 
--   `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
+       - `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
 
-2. Create a persistent volume with a disk format on a user specified datastore.
+    2. Create a persistent volume with a disk format on a user specified datastore.
+  
+        ```yaml
+        kind: StorageClass
+        apiVersion: storage.k8s.io/v1
+        metadata:
+          name: fast
+        provisioner: kubernetes.io/vsphere-volume
+        parameters:
+            diskformat: zeroedthick
+            datastore: VSANDatastore
+        ```
 
-```yaml
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: fast
-provisioner: kubernetes.io/vsphere-volume
-parameters:
-    diskformat: zeroedthick
-    datastore: VSANDatastore
-```
+       -   `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
+       -   `datastore`: The user can also specify the datastore in the StorageClass. The volume will be created on the datastore specified in the storage class which in this case is `VSANDatastore`. This field is optional. If the datastore is not specified, then the volume will be created on the datastore specified in the vSphere config file used to initialize the vSphere Cloud Provider.
 
--   `diskformat`: `thin`, `zeroedthick` and `eagerzeroedthick`. Default: `"thin"`.
--   `datastore`: The user can also specify the datastore in the Storageclass. The volume will be created on the datastore specified in the storage class which in this case is `VSANDatastore`. This field is optional. If not specified as in previous YAML description, the volume will be created on the datastore specified in the vsphere config file used to initialize the vSphere Cloud Provider.
+    3. Storage Policy Management inside kubernetes
 
-3. Create a persistent volume with user specified VSAN storage capabilities.
+      - Using existing vCenter SPBM policy
 
-```yaml
-kind: StorageClass
-apiVersion: storage.k8s.io/v1
-metadata:
-  name: vsan-policy-fast
-provisioner: kubernetes.io/vsphere-volume
-parameters:
-  diskformat: thin
-  hostFailuresToTolerate: "1"
-  diskStripes: "2"
-  cacheReservation: "20"
-  datastore: VSANDatastore
-```
+        One of the most important features of vSphere for Storage Management is Policy based Management. Storage Policy Based Management (SPBM) is a storage policy framework that provides a single unified control plane across a broad range of data services and storage solutions. SPBM enables vSphere administrators to overcome upfront storage provisioning challenges, such as capacity planning, differentiated service levels and managing capacity headroom
```

Comment:
In the first sentence, "policy based management" should all be in lowercase since it's not a particular name. The capitalization in the second sentence is okay because SPBM is the name of the framework.

Reply:
Fixed!